CMakeLists.txt

```diff
@@ -118,11 +121,6 @@ if(ENABLE_ARES)
   set(CURL_LIBS ${CURL_LIBS} ${CARES_LIBRARY})
 endif()
 
-if(MSVC)
-  option(BUILD_RELEASE_DEBUG_DIRS "Set OFF to build each configuration to a separate directory" OFF)
-  mark_as_advanced(BUILD_RELEASE_DEBUG_DIRS)
-endif()
-
```

Comment:
Because I could not see anywhere that the flag was used.
Its not a standard CMake flag (as far as I know), and its not mentioned anywhere else in the Curl code.